### PR TITLE
Fix link fragments in Heroku deployment guide

### PR DESCRIPTION
**Pull request by shagedorn**
Minor follow-ups of #472 that were detected during testing.

### CODE REVIEW

File: server/guides/deploying/heroku.md [hunk 1]
```diff
@@ -136,4 +136,4 @@ git push heroku main
 
 This will take significantly longer than regular git operations as Heroku will build your project incl. all dependencies from scratch, verify and deploy it.
 
-More information can be found on [here](https://devcenter.heroku.com/articles/git#deploying-code).
+More information can be found on [here](https://devcenter.heroku.com/articles/git#deploy-your-code).
```
Comment (shagedorn) [inline server/guides/deploying/heroku.md:139]: This might be prone to changes… shall we leave out the fragment for a more stable link?

Comment (0xTim) [reply server/guides/deploying/heroku.md:139]: I'm not too concerned about it, worst case is that it just navigates to the page as if the fragment wasn't provided in the first place

**Review by 0xTim — APPROVED** [review]
Thanks!

**Comment by 0xTim** [comment]
@swift-ci please test

**Pull request merged**